src/DataSource.test.ts

```diff
@@ -0,0 +1,85 @@
+import { DataSource } from './DataSource';
+import { DataSourceInstanceSettings, PluginMeta, ScopedVars, TypedVariableModel } from '@grafana/data';
+import { QueryType, SitewiseOptions, SitewiseQuery } from './types';
+import { assetIdVariableArray, assetIdVariableConstant } from './__mocks__/variableMocks';
+
+const testInstanceSettings = (
+  overrides?: Partial<DataSourceInstanceSettings<SitewiseOptions>>
+): DataSourceInstanceSettings<SitewiseOptions> => ({
+  id: 1,
+  uid: 'sitewise-test',
+  type: 'sitewise',
+  name: 'sitewise-test',
+  meta: {} as PluginMeta,
+  readOnly: false,
+  jsonData: {} as SitewiseOptions,
+  access: 'direct',
+  ...overrides,
+});
+
+jest.mock('@grafana/runtime', () => ({
+  ...jest.requireActual('@grafana/runtime'),
+  getTemplateSrv: () => {
+    return {
+      getVariableName(variableId: string) {
+        return variableId.slice(1);
```

Comment:
I think I don't follow this slice, is it just generating a string? Would it work the same as if we returned the variableId?

Reply:
Youre totally right, this was initially gonna be a more complicated test so this can definitely be simplified now! Just pushed